src/components/Dialog/Dialog.test.tsx

```diff
@@ -0,0 +1,89 @@
+import { fireEvent, render } from '@testing-library/react';
+import { Dialog } from './Dialog';
+
+
+const MockComponent = () => {
+	return <div tabIndex={0}>
+		<div tabIndex={1} data-testid="mock-component">
+			Mock Component
+		</div>
+	</div>;
+};
+
+xdescribe('Dialog component tests', () => {
```

Comment:
I'm getting an error: Your focus-trap must have at least one container with at least one tabbable node in it at all times portal